src/components/Hero/AttributesBlock.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { shape, number, string, bool } from 'prop-types';
+import styled from 'styled-components';
+import strings from 'lang';
+import AttrStrength from 'components/Icons/AttrStrength';
+import AttrAgility from 'components/Icons/AttrAgility';
+import AttrIntelligent from 'components/Icons/AttrIntelligent';
+import constants from 'components/constants';
+import Attribute from './Attribute';
+
+
+const AttributesWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+  flex-wrap: wrap;
+`;
+
+const AttributeBlock = styled.div`
+  &:last-child {
+    margin-right: initial;
+  }
+
+  display: flex;
+  flex-direction: column;
+  margin-top: 10px;
+`;
+
+const Label = styled.span`
+  margin-right: 5px;
+  color: ${constants.colorMutedLight};
+`;
+
+const renderIcon = Icon => <Icon style={{ width: 20, marginRight: 4 }} />;
+
+const calcArmorPercent = hero => Math.round(0.06 * hero / (1 + (0.06 * hero)) * 100);
```

Comment:
I'm not quite sure what this is, can you annotate/comment it?

Reply:
Yes, sure